br/pitr-usage.md

```diff
@@ -23,7 +23,7 @@ summary: 了解如何使用 PiTR。
 | TiDB | 8 核+ | 16 GB+ | SAS | c5.2xlarge | 2 |
```

Comment:
This file is aligned in https://github.com/pingcap/docs/pull/9823/commits/0b175c28c93723603b3bcf2956793bd84f5da335